dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -49,6 +49,7 @@ const TYPE_OCI = "oci";
 
 export function AppRepoForm(props: IAppRepoFormProps) {
   const { onSubmit, onAfterInstall, namespace, kubeappsNamespace, repo, secret } = props;
+  const workingRef = useRef(false);
```

Comment:
I'd maybe choose another name, a more self-explanatory one like  `isInstallingRef` ?

Reply:
You are right, will rename it. Initially I wanted to make the solution something general for all forms+buttons :)